mule-gateway/modules/ROOT/pages/mule-gateway-proxy-config.adoc

```diff
@@ -0,0 +1,95 @@
+= Connect the Mule Gateway Through a Proxy Server
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+If your Mule runtime engine runs inside a firewall that restricts external communication through a proxy server, you can configure the Mule Gateway to route traffic through the proxy server to API Manager.
+
+The Mule Gateway supports the Basic, NTLM and Kerberos v5 (via SPNEGO) authentication schemes for proxied communication.
+
+
+== Configure the Mule Gateway to Connect Through a Proxy Server
+
+To configure the Mule Gateway to connect through a proxy server, you must set up your proxy server configuration in the `$MULE_HOME/conf/wrapper.conf` file.
+
+
+=== Set Up Proxy Server Configuration in the wrapper.conf File
+
+To specify proxy server configuration in the `$MULE_HOME/conf/wrapper.conf` file,
+add your proxy server information to the following properties:
+
+* `wrapper.java.additional.<n>=-Danypoint.platform.proxy_host={hostname}`
+* `wrapper.java.additional.<n>=-Danypoint.platform.proxy_port={port}`
+* `wrapper.java.additional.<n>=-Danypoint.platform.proxy_username={username}`
+* `wrapper.java.additional.<n>=-Danypoint.platform.proxy_password={password}`
+
+[IMPORTANT]
+These are additional parameters to pass to Java when it is launched. The <n> element refers to the number of additional parameters in the configuration. It is indicated with an integer number counting up from `1` and must follow a sequence without any gaps. 
+
+With the credentials provided, Mule Gateway will prefer Basic Authentication over NTLM. Failure to authenticate via Basic Auth would make Gateway attempt login with NTLM. For Kerberos, extra properties are required that are explained in the next section.
+
+=== Using Kerberos for Proxy Authentication
+
+Alongside the properties described in the previous section, add the following properties:
+* `wrapper.java.additional.<n>=-Danypoint.platform.proxy_kerberos_krb5_path={/path/to/krb5.conf}`
+* `wrapper.java.additional.<n>=-Danypoint.platform.proxy_kerberos_jaas_path={/path/to/kerberos.jaas}`
+
+NOTE: Kerberos authentication is only supported in Mule runtime versions 4.3.x and 4.4.x released after July 2, 2024 and Mule runtime versions 4.6.5, 4.7.1 and later.
```

Comment:
```suggestion
NOTE: Kerberos authentication is only supported in Mule runtime versions 4.3.x and 4.4.x, and Mule runtime versions 4.6.5, 4.7.1 and later.
```

Reply:
In this we should keep that `released after July 2, 2024`. 4.3.x and 4.4.x mule distributions have a timestamp that distinguish when the patch was released. They don't follow the patch increment and 4.6.x and 4.7.x that go onto 4.6.5 and 4.7.1 respectively. So it's important to keep the July 2 for customers to know as from which 4.3.x/4.4.x release the feature is available